SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -205,7 +206,7 @@ namespace SpatialConstants
 	const WorkerRequirementSet ClientOrServerPermission{ {UnrealClientAttributeSet, UnrealServerAttributeSet} };
 
 	static const FString ClientsStayConnectedURLOption = TEXT("clientsStayConnected");
-	static const FString SnapshotURLOption = TEXT("snapshot=");
+	static const FString SpatialSessionIdURLOption = TEXT("spatialSessionId=");
```

Comment:
Think `"spatialSessionId="` can be `"spatialSessionId"` if we parse URL options correctly.

Reply:
Could not get this to work.